VCE/gather_devices.py

```diff
@@ -200,7 +205,9 @@ def logout(logout_session, x_verkada_token, x_verkada_auth, org_id=ORG_ID):
 ##############################################################################
 
 
-def list_cameras(api_key, camera_session):
+def list_cameras(
+    api_key: str, camera_session: requests.Session
+) -> Optional[List[str]]:
```

Comment:
Keeping the statements checking for 403 and 400 as they are used for custom logging. They typing hints have been modified along with the return types. Removed devices_serials as it was unused. The try-except will remain the way it is since that is the only function that has thrown the error in the past.